www/pending-report

```diff
@@ -0,0 +1,84 @@
+<?php
+
+include_once "util.php";
+include_once "pagetpl.php";
+include_once "dbconnect.php";
+
+$db = dbConnect();
+
+$result = mysqli_execute_query($db, "select gameid, title, url from gamelinks where url like '%ifarchive-pending%'");
+
+/*
+there since version N, still there: there's a latest links entry for pagevsn N-1 which does NOT contain the link
+there since version 1: there's not a links entry that does NOT contain the link
+    either because there are no links entries
+    or because all of the links entries have it
+*/
+
+$results = [];
+
+while ($row = mysqli_fetch_row($result)) {
+    list($gameid, $file, $url) = $row;
```

Comment:
All the `list(...)` stuff can be replaced with `[...]` - I think it's more readable.